packages/users/rainbow/fee-collector/test/RainbowClaimer.test.ts

```diff
@@ -0,0 +1,355 @@
+import {
+  assertEvent,
+  assertIndirectEvent,
+  assertNoEvent,
+  bn,
+  deploy,
+  deployProxy,
+  deployTokenMock,
+  fp,
+  getSigners,
+  NATIVE_TOKEN_ADDRESS,
+  ZERO_ADDRESS,
+  ZERO_BYTES32,
+} from '@mimic-fi/v3-helpers'
+import { buildEmptyTaskConfig, deployEnvironment } from '@mimic-fi/v3-tasks'
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { expect } from 'chai'
+import { BigNumberish, Contract, ethers } from 'ethers'
+import { ethers as ethershh } from 'hardhat'
+
+describe('RainbowClaimer', () => {
+  let task: Contract, smartVault: Contract, authorizer: Contract, feeCollector: Contract
+  let owner: SignerWithAddress, other: SignerWithAddress
+
+  before('setup', async () => {
+    // eslint-disable-next-line prettier/prettier
+    [, owner, other] = await getSigners()
+    ;({ authorizer, smartVault } = await deployEnvironment(owner))
+  })
+
+  beforeEach('deploy task', async () => {
+    feeCollector = await deploy('FeeCollectorMock')
+    task = await deployProxy(
+      'RainbowClaimer',
+      [],
+      [buildEmptyTaskConfig(owner, smartVault), feeCollector.address],
+      'initializeRainbowClaimer'
+    )
+  })
+
+  describe('execution type', () => {
+    it('defines correctly', async () => {
+      const expectedType = ethers.utils.solidityKeccak256(['string'], ['RAINBOW_CLAIMER'])
+      expect(await task.EXECUTION_TYPE()).to.be.equal(expectedType)
+    })
+  })
+
+  describe('setFeeClaimer', () => {
```

Comment:
Let's fix all the occurrences

```suggestion
  describe('setFeeCollector', () => {
```